api/serializers.py

```diff
@@ -113,6 +114,27 @@ def to_representation(self, instance):
         return representation
 
 
+class CommentLikeSerializer(serializers.ModelSerializer):
+    parent_lookup_kwargs = {
+        'author_pk': 'author__pk',
+        'post_pk': 'post__pk',
```

Comment:
```suggestion
```
I think this can be taken out since we're not querying for the post (but leave author_pk in, we're still querying the author)